packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -156,7 +156,7 @@ abstract class FlutterGoldenFileComparator extends GoldenFileComparator {
     return goldenFile;
   }
 
-  /// Prepends the golden Uri with the library name that encloses the current
+  /// Prepends the golden URI with the library name that encloses the current
```

Comment:
URL

You should ban URI from your dictionary

Reply:
OK, done.  And replaced all (reasonable) instances of URI with URL in the other places where they appeared.